core/src/plugins/outputs/panels/javascript.tsx

```diff
@@ -19,10 +19,10 @@ export function JSPanel({ UI, devtoolsWindow: { simport } }: ReactRenderProps) {
 
       return files
         .filter(({ name }) => name.endsWith('.js'))
-        .map(({ name, text, editorText }) => `// @filename:${name}\n${
+        .map(({ name, text, editorText, tsCompilerResultText }) => `// @filename:${name}\n${
           editorText.match(/^\/\/ @devtools.output.compiled\r?\n/)
-            ? text
-            : editorText
+            ? tsCompilerResultText
+            : text
```

Comment:
```suggestion
            ? text
            : tsCompilerResultText
```

应该是反了，有 @devtools.output.compiled 标记的时候展示 babel 的编译结果，没有的时候展示 tsc 的编译结果